yaas/parsers/service.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+
+from .. import service
+from .. import config
+from .. import utils
+
+def define_subcommand(parent_parser):
```

Comment:
The terminology I'm looking for is "[pure function](https://en.wikipedia.org/wiki/Pure_function)".
My first example is pure, the second one is not (and this usage in this PR is not).

But ya... it's minor.